Libraries/LibLine/XtermSuggestionDisplay.cpp

```diff
@@ -62,8 +64,45 @@ void XtermSuggestionDisplay::display(const SuggestionManager& manager)
         lines_used += max_line_count;
         longest_suggestion_length = 0;
     }
+
     VT::move_absolute(max_line_count + m_origin_x, 1);
 
+    if (m_pages.size() == 0) {
```

Comment:
[Please use](https://rust-lang.github.io/rust-clippy/master/#len_zero) `is_empty()` instead of checking for `size() == 0`.

Reply:
Does this really make sense in this context? we know that `Vector::size()` is just a member access, inline at that...

Applied nonetheless.